packages/e2e/cypress/cli/dbt/cli.cy.ts

```diff
@@ -347,4 +349,24 @@ describe('CLI', () => {
                 );
         });
     });
+
+    it('Should test validate', () => {
```

Comment:
Since this command also depends on dbt, we need to do  it with dbt dependencies, so we can't use the `commands.cy.ts` which runs only once